deps: update modeling dependencies

smbea opened the pull request.

smbea commented on CHANGELOG.md:
```diff
@@ -34,7 +41,7 @@ ___Note:__ Yet to be released changes appear here._
 ## 1.5.0
 
 * `FEAT`: allow external variable providers to be added ([variable-resolver#1](https://github.com/bpmn-io/variable-resolver/pull/1))
-* `DEPS`: update to `bpmn-js@1.19.1`
+* `DEPS`: update to `bpmn-js@11.5.1`
```
Typo in previous changelog update

smbea commented on CHANGELOG.md:
```diff
@@ -34,7 +41,7 @@ ___Note:__ Yet to be released changes appear here._
 
 * `FEAT`: allow external variable providers to be added ([variable-resolver#1](https://github.com/bpmn-io/variable-resolver/pull/1))
-* `DEPS`: update to `bpmn-js@1.19.1`
+* `DEPS`: update to `bpmn-js@11.5.1`
```
Yeah this was a small typo I noticed. Likely came from copying the `bpmn-js-properties-panel` version bellow

marstamm commented:
Looks good to me, note that @nikku will cut a patch release for `bpmn-js-properties-panel` shortly. Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/902

smbea commented:
Fixed conflicts and updated changelog version already with https://github.com/camunda/camunda-bpmn-js/pull/267/commits/dc636a8dd310c019d1d0c4bdd13ee8878b1cf48d

marstamm reviewed (approved):
🚀
